proofs.md

```diff
@@ -24,20 +24,19 @@ Throughout this document, the following definitions are used:
 - __*original data:*__ the concatenation of a __*sector's*__ constitutent pieces, all __*piece padding*__, and any __*terminal padding*__.
-- __*unsealed sector:*__ a concrete representation (on disk or in memory) of a sector's __*original data*__. 
-- __*sealed sector:*__ a concrete representation (on disk or in memory) of the unique replica generated by `Seal` from an __*unsealed sector*__. 
+- __*unsealed sector:*__ a concrete representation (on disk or in memory) of a sector's __*original data*__.
+- __*sealed sector:*__ a concrete representation (on disk or in memory) of the unique replica generated by `Seal` from an __*unsealed sector*__.
 - __*piece padding:*__ a block of zero or more 'zero bytes' inserted between __*pieces*__ to ensure they are positioned within the containing __*sector*__ in a way compatible with the __*Piece Inclusion Proof*__.
 - __*terminal padding:*__ a block of zero or more 'zero bytes' inserted after a __*sector's*__ final piece, ensuring that the length of the __*original data*__ is `SECTOR_BYTES`.
 - __*preprocessing:*__ a transformation applied to an __*unsealed sector*__ as the first stage of sealing and which may increase the size of the data.
 - __*preprocessed data:*__ the result of __*preprocessing*__ the __*original data*__.
 - __*preprocessed sector:*__ a concrete representation (on disk or in memory) of a __*sector's*__ __*preprocessed data*__.
 - __*SNARK proof:*__ a block of bytes which proves that the creator is in possession of a satisfying assignment to a quadratic arithmetic program (__*circuit*__).
-- __*Multi-SNARK proof:*__ a block of one or more __*SNARK proofs*__, each proving partition of the total set of challenges to be proved.  
+- __*Multi-SNARK proof:*__ a block of one or more __*SNARK proofs*__, each proving partition of the total set of challenges to be proved.
 - __*Merkle inclusion proof:*__ a proof (whose format is unspecified here) that a given leaf block is contained within a merkle tree whose root is the __*commitment*__ associated with the proof.
 - __*commitment:*__ an opaque block of data to which a prover 'commits', enabling subsequent proofs which cannot be validly constructed unless the __*commitment*__ itself was validly constructed. For example: the output of a suitable pseudorandom collision-resistant hash function may serve as a __*commitment*__ to the data which is the preimage of that hash. Publication of the __*commitment*__ proves that the creator was in possession of the preimage at the time the __*commitment*__ was generated.
 - __*prover:*__ the party who generates a proof, in Filecoin it's always the Storage Miner.
 - __*verifier:*__ the party who verifies a proof generated by a __*prover*__, in Filecoin it's a full node.
-- __*sectors count:*__ the number of sectors over which a proof-of-spacetime is performed (`POST_SECTORS_COUNT`, `16 PiB / 64 GiB = 262144`).
```

Comment:
Does eliminating `POST_SECTORS_COUNT` mean that we can cap the number of sectors which a miner can commit to the network to MAX(uint64)? If so, would you please update the `actors.md` `CommitSector` method? It checks `if miner.Sectors.Size() >= POST_SECTORS_COUNT { ... fail` now.

Reply:
done